lib/ansible/plugins/action/eos.py

```diff
@@ -79,7 +79,7 @@ def run(self, tmp=None, task_vars=None):
                 # enable mode and not config module
                 rc, out, err = connection.exec_command('prompt()')
                 while str(out).strip().endswith(')#'):
-                    display.debug('wrong context, sending exit to device', self._play_context.remote_addr)
+                    display.debug('<%s> wrong context, sending exit to device' % self._play_context.remote_addr)
```

Comment:
OK, if we are being consistent with something else that makes sense.

I personally find putting the message in quotes can be useful for checking for trailing whitespace, , as well as making the message stand out more.